app/frontend/css/index.js

```diff
@@ -96,6 +100,8 @@ document.addEventListener('DOMContentLoaded', (event) => {
         schemeSIG.checked = true
         schemeSFI.checked = true
       }
+
+      trackConversationEvent && trackConversationEvent(`scheme was filtered to ${schemeAll.checked ? 'enable' : 'disable'} all`)
```

Comment:
Since we've removed the pill filters, these can be removed. Could clean up additional code here regarding checking of scheme buttons